src/docs/templates/our-story.hbs

```diff
@@ -47,7 +47,7 @@
       </div>
       <figcaption class="col-12 col-lg-4">
         <div class="imagewtext__content imagewtext--reverse text-left mt-lg-0 p-2 pt-lg-3 pb-lg-3 pl-lg-3">
-          <h1>We've spent years testing, developing and perfecting our formulas</h1>
+          <h1 class="mb-2">We've spent years testing, developing and perfecting our formulas</h1>
```

Comment:
please use only one H1 per page, usually the top title, the rest can take h1 class